src/main/java/com/example/rets_api/resource/Enums.java

```diff
@@ -78,4 +78,19 @@ public enum Operator {
         DEFAULT_ENUM_VALUE
     }
 
+    public enum CommunityType {
```

Comment:
You can remove this type because is not used anymore

Reply:
done